README.md

```diff
@@ -44,3 +44,82 @@ along with this program.  If not, see https://www.gnu.org/licenses/
 ___
 
 Project Website: https://metrocs.github.io/imagelab/
+
```

Comment:
Adding the new section immediately below leaves the "Project Website" reference just hanging.
Consider how all the parts fit together to provide a cohesive organization and appearance.